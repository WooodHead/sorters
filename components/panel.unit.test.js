import Panel from './panel'
import renderer from 'react-test-renderer'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'

describe('panel', () => {
    it('shallow-renders', () => {
        const component = shallow(<Panel
            title="Title"
            url="/foo"
            label="Foo"
        ><p>Hi</p></Panel>)
        const tree = toJson(component)
        expect(tree).toMatchSnapshot()
    })
})
