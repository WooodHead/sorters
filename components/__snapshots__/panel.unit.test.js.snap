// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`panel shallow-renders 1`] = `
<div
  className="panel panel-default"
>
  <div
    className="panel-heading"
  >
    <h3
      className="panel-title"
    >
      <a
        href="/foo"
      >
        Title
      </a>
    </h3>
  </div>
  <div
    className="panel-body"
  >
    <p>
      Hi
    </p>
    <a
      className="btn btn-default"
      href="/foo"
      role="button"
    >
      Foo
    </a>
  </div>
</div>
`;
